pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -475,9 +475,8 @@ type LinuxSandboxSecurityContext struct {
 	NamespaceOptions *NamespaceOption `protobuf:"bytes,1,opt,name=namespace_options,json=namespaceOptions" json:"namespace_options,omitempty"`
 	// Optional SELinux context to be applied.
 	SelinuxOptions *SELinuxOption `protobuf:"bytes,2,opt,name=selinux_options,json=selinuxOptions" json:"selinux_options,omitempty"`
-	// User to run the entrypoint of the sandbox process. Can be either UID or
-	// user name.
-	RunAsUser *string `protobuf:"bytes,3,opt,name=run_as_user,json=runAsUser" json:"run_as_user,omitempty"`
+	// UID to run sandbox processes as, when applicable.
```

Comment:
@liggitt this is about the `PodSandboxConfig` user which is the `pause` container (and only the pause container), not the user of the actual application that makes up said daemonset.